optax/_src/loss.py

```diff
@@ -503,3 +503,19 @@ def kl_divergence_with_log_targets(log_predictions: chex.Array,
   chex.assert_type([log_predictions, log_targets], float)
   loss = jnp.exp(log_targets) * (log_targets - log_predictions)
   return jnp.sum(loss, axis=-1)
+
+def hinge_loss(predictions: chex.Array, targets: chex.Array) -> chex.Array:
+  """
+  Computes the hinge loss for Binary Classification.
+
+  Args:
+    predictions: Predicted values with shape [..., dim].
+    targets: Target values with shape [..., dim].
+      Expected to be in range [-1, 1].
```

Comment:
I think they have to be strictly in the set {-1, 1}, i.e. either -1 or 1, not in the interval [-1, 1]. Could you clarify this?